setup.py

```diff
@@ -27,6 +27,7 @@ def read(fname):
     url="https://github.com/TimDettmers/bitsandbytes",
     packages=find_packages(),
     package_data={"": libs},
+    install_requires=['torch', 'numpy', 'scipy', 'pandas', 'matplotlib'],
```

Comment:
`pandas` and `matplotlib` are only required for the benchmarking scripts; they should be required only as extras:
```suggestion
    install_requires=['torch', 'numpy', 'scipy'],
    extras_require={'benchmark': ['pandas', 'matplotlib']},
```